@uportal/esco-content-menu/src/components/ContentGrid.vue

```diff
@@ -227,7 +227,7 @@ export default {
                 threshold: matchSorter.rankings.ACRONYM
               });
 
-      return valueFilter(categoryFilter(portlets));
+      return valueFilter(categoryFilter(portlets)).sort(byPortlet);
```

Comment:
@bjagg is there an example of this issue?
Testing locally I'm not seeing a change having this vs not having it, but I'm probably just not looking in the right place.

Reply:
@ChristianMurphy It was challenging to duplicate the issue. I finally received some client data that highlighted the issue. I can share it privately. It's a corner case where only one category was out of order.

Reply:
I'm not sure this produces the desired behavior.
searching for a portlet including "ges", without this change:
![search-when-disabled](https://user-images.githubusercontent.com/3107513/82687777-9a456080-9c0c-11ea-9ef7-0e3295090772.gif)
notice that all the top results have an exact match.

Searching for a portlet with "ges", with this change:
![search-when-enabled](https://user-images.githubusercontent.com/3107513/82687898-ccef5900-9c0c-11ea-91ea-ebf33ada397e.gif)

Note that "flash info guest" is promoted to the top result, even though it is a less accurate match.

Reply:
I'm not sure this produces the desired behavior.

searching for a portlet including "ges", without this change:
![search-when-disabled](https://user-images.githubusercontent.com/3107513/82687777-9a456080-9c0c-11ea-9ef7-0e3295090772.gif)
notice that all the top results have an exact match.

Searching for a portlet with "ges", with this change:
![search-when-enabled](https://user-images.githubusercontent.com/3107513/82687898-ccef5900-9c0c-11ea-91ea-ebf33ada397e.gif)

Note that "flash info guest" is promoted to the top result, even though it is a less accurate match.

Reply:
From an out of band discussion, it appears `categoryFilter` can in some cases disturb the pre-existing sort of `portlets`.
Changing this to `valueFilter(categoryFilter(portlets.sort(byPortlet)))` so that category filter will remain sorted, and `valueFilter` can still rank appears to resolve the issue, and allow for the other `sort(byPortlet)` in this file to be dropped.